scripts/local_cloudbuild.py

```diff
@@ -274,7 +288,9 @@ def local_cloudbuild(args):
     # Run shell script
     if cloudbuild.run:
         args = [os.path.abspath(cloudbuild.output_script)]
-        subprocess.check_call(args)
+        return subprocess.check_output(args)
```

Comment:
Good catch.  Restored the previous behavior, captured output in the test code instead.